test/ejabberd_tests/full.spec

```diff
@@ -16,6 +16,7 @@
 {suites, "tests", login_SUITE}.
 {suites, "tests", muc_SUITE}.
 {suites, "tests", mam_SUITE}.
+{suites, "tests", oauth_SUITE}.
```

Comment:
In order to use this SUITE in travis build, `default.spec` file has to be modified as well.